bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -281,11 +288,10 @@ impl<
 					.into(),
 			},
 			RefundSurplus,
```

Comment:
In case of an error, It's better to put this inside of the appendix. Do refund and then deposit all

Reply:
good point, done